Jak-Sidious/AndroidLevelUp
# Ft instapk and configuration changes #156005187

#### What does this PR do?
Handle the change of configuration i.e change of screen rotation.
#### Description of Task to be completed?
Currently, when the user rotates screen, all data would be lossed, but with the changes in this PR, on screen rotation, the data shall be viewable in both portrait and landscape modes.
#### How should this be manually tested?
The repo can be cloned and the application run. On running, the tester can rotate the screen and be able to see that the data remains viewable in both modes.
#### Any background context you want to provide?
None
#### What are the relevant pivotal tracker stories?
#156005187
#### Screenshots (if appropriate)
**Landscape View**
<img width="726" alt="screen shot 2018-05-29 at 10 19 02" src="https://user-images.githubusercontent.com/30991429/40643800-d6abf408-6329-11e8-981b-60b6be3153e8.png">

**Portrait View**
<img width="354" alt="screen shot 2018-05-29 at 10 19 32" src="https://user-images.githubusercontent.com/30991429/40643820-e8f57206-6329-11e8-9686-f4ac815ac914.png">


#### Questions:
